src/main/java/net/minecraftforge/common/util/Constants.java

```diff
@@ -174,7 +174,7 @@
     public static class AiMutexBits {
         public static final int MOTION  = 0b001;
         public static final int LOOKING = 0b010;
-        public static final int MIST    = 0b100;
+        public static final int JUMP    = 0b100;
```

Comment:
Three words should all be verbs or nouns, e.g. `MOVE` `LOOK` `JUMP` and `MOTION` `LOOKING` `JUMPING`

Reply:
I think you mean the present participle `MOVING`, as `LOOKING` and `JUMPING` are both present participles, while `MOTION` is an uncountable noun.